### ISSUE TITLE
Importing the Dialog SCSS affects other form controls

### ISSUE THREAD

**Issue by svetq** [telerik/kendo-themes#1484]
**Describe the bug**
When importing the required SCSS per component in the root styles.scss file:

> @import '~@progress/kendo-theme-bootstrap/scss/datetime/_index.scss';

>  @import '~@progress/kendo-theme-bootstrap/scss/numerictextbox/_index.scss';

and using the following markup:

```
<kendo-numerictextbox class="form-control">
</kendo-numerictextbox>

<kendo-datepicker class="form-control">
</kendo-datepicker>

```
The appearance is fine:

![image](https://user-images.githubusercontent.com/20015190/79457340-c01d8b00-7ff8-11ea-9f42-815c0f5d00ea.png)

But adding the Dialog SCSS:

> @import '~@progress/kendo-theme-bootstrap/scss/dialog/_index.scss';

causes the NumericTextBox and DateTimePicker to change their width:

![image](https://user-images.githubusercontent.com/20015190/79457328-b7c55000-7ff8-11ea-87fa-f7073f81fed2.png)

Please check the attached project:
[test-scss.zip](https://github.com/telerik/kendo-themes/files/4487036/test-scss.zip)

**Affected package (please remove the unneeded items)**
* theme-default
* theme-bootstrap
* theme-material

**Affected suites (please remove the unneeded items)**
* Kendo UI for Angular

**Affected browsers (please remove the unneeded items)**
* All

**Build system information (please remove the unneeded items)**
* Not Applicable

**Comment by joneff** [comment]
As with many of our components, we include, what we consider to be, dependent files, so developers don't have to include them on their own. For instance, the grid styles will include the styles for toolbar, inputs, button, pager etc.

Such is the case with the dialog, which includes our forms module, because it is expected to layout forms inside the window. That same module includes a compatibility layer for bootstrap (or the so called bootstrap forms integration), which massages our components to integrate better with bootstrap form classes. One such integration is respecting the width of `.form-control`, which happens to be 100%.

In short, having a class `form-control` on your inputs and bootstrap loaded (regardless of the theme) will most certainly affect the width of our components.

Considering is is the desired effect, I am closing the issue.
